lib/tasks.js

```diff
@@ -271,6 +271,19 @@ module.exports = async options => {
       await run(`!rm -rf ${dist.stageDir}`);
     }
 
+    if (github.release) {
+      getSpinner().stopAndPersist({
+        symbol: `❗`,
+        text: `To Github repository ${config.options.repo.remote}`
+      });
+    }
+    if (npm.publish) {
+      getSpinner().stopAndPersist({
+        symbol: `❗`,
+        text: `To NPM package ${config.options.npm.packageUrl}`
+      });
```

Comment:
Something like this should be enough here:

```
log(`🔗 ${npm.getPackageUrl()}`);
```

Reply:
Okay.